persistence/sql/persister_consent.go

```diff
@@ -85,7 +83,7 @@ func (p *Persister) revokeConsentSession(whereStmt string, whereArgs ...interfac
 }
 
 func (p *Persister) RevokeSubjectLoginSession(ctx context.Context, subject string) error {
-	if err := p.Connection(ctx).RawQuery("DELETE FROM hydra_oauth2_authentication_session WHERE subject = ?", subject).Exec(); errors.Is(err, sql.ErrNoRows) {
+	if err := p.Connection(ctx).RawQuery("DELETE FROM hydra_oauth2_authentication_session WHERE subject = ? AND nid = ?", subject, p.NetworkID(ctx)).Exec(); errors.Is(err, sql.ErrNoRows) {
```

Comment:
Does the original (as well as updated) code do what it purports to do? I don't think that the code can result in `ErrNoRows`. The only way to report ErrNotFound that I'm aware of is to use ExecWithCount, which is incompatible with QueryWithNetwork.

Reply:
I see that this was discussed in #2397. I'll remove the not found error handling because I couldn't reproduce the error, even after testing all supported databases.